### ISSUE TITLE
PROPN in multiword tokens

### ISSUE THREAD

**Issue by kanayamah** [UniversalDependencies/UD_German-GSD#9]
Generally German multiword tokens are combination of ``ADP`` and ``DET`` (_e.g._ "zur" and "beim"), but I found ``PROPN`` and ``DET`` are tagged for such tokens embedded in proper nouns.  For example:
```
18	Karlsson	Karlsson	PROPN	NN	Case=Acc|Gender=Masc|Number=Sing	22	obj	_	_
19-20	vom	_	_	_	_	_	_	_	_
19	von	von	PROPN	APPR	_	21	case	_	_
20	dem	der	DET	ART	Case=Dat|Definite=Def|Gender=Masc,Neut|Number=Sing|PronType=Art	21	det	_	_
21	Dach	Dach	PROPN	NN	Case=Dat|Gender=Masc,Neut|Number=Sing	18	nmod	_	_
```

If it is a part of ``PROPN``, both of "von" and "dem" should be consistently tagged as ``PROPN``, and if the original PoS should be tagged in decomposed words, "von" should be ``ADP`` as long as "dem" is ``DET``, shouldn't they?  
There are more than 50 cases in the training corpus.

**Comment by dan-zeman** [comment]
Thanks, Hiroshi, for the report. This is a systematic bug that has appeared in several treebanks converted from Google data, including German. All words in multi-word named entities are tagged `PROPN`, regardless their real part of speech; this is a violation of the UD guidelines.

In your example, _von_ should be tagged `ADP`. Even _Dach_ should be `NOUN` and not `PROPN`.

**Comment by kanayamah** [comment]
Thank you, Dan.  Waiting for the fix.

**Comment by dan-zeman** [comment]
Fixed in a5cd3e29a19a7ae07990acaaa742c18523133df5.

**Comment by AngledLuffa** [comment]
Awesome, thanks
